src/utils/data_reader.py

```
@@ -0,0 +1,136 @@
+        end_date: datetime
+    ) -> pd.DataFrame:
+        files = self.manager.list_files('era5', 'era5_pbl_*.csv')
+        
+        relevant_files = []
+        for file in files:
+            parts = file.stem.split('_')
+            if len(parts) >= 4:
+                try:
+                    year = int(parts[2])
+                    month = int(parts[3])
+                    file_date = datetime(year, month, 1)
```

Ruff (0.12.2)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)